corehq/apps/style/templates/style/base.html

```diff
@@ -193,9 +198,19 @@
         {% block head %}
         {% endblock %}
 
+        {% if not request.use_requirejs %} {# TODO: requirejs support #}
         {% include 'style/includes/analytics_all.html' %}
+        {% endif %}
+
+        <script type="text/javascript">
+            window.USE_REQUIREJS = {% if request.use_requirejs %}true{% else %}false{% endif %};
```

Comment:
Needed in addition to `request.use_requirejs` for the sake of report filters, which don't have access to the request when they're rendered.